ratbagd/ratbagd-button.c

```diff
@@ -310,6 +310,48 @@ static int ratbagd_button_set_macro(sd_bus *bus,
 	return 0;
 }
 
+static int ratbagd_button_get_none(sd_bus *bus,
+				   const char *path,
+				   const char *interface,
+				   const char *property,
+				   sd_bus_message *reply,
+				   void *userdata,
+				   sd_bus_error *error)
+{
+	CHECK_CALL(sd_bus_message_append(reply, "(uv)",
+					 RATBAG_BUTTON_ACTION_TYPE_NONE,
+					 "u",
+					 0));
+
+	return 0;
+}
+
+static int ratbagd_button_set_none(sd_bus *bus,
+				   const char *path,
+				   const char *interface,
+				   const char *property,
+				   sd_bus_message *m,
+				   void *userdata,
+				   sd_bus_error *error)
+{
+	struct ratbagd_button *button = userdata;
+	int r;
+	_unused_ unsigned int zero;
+
+	CHECK_CALL(sd_bus_message_read(m, "v", "u", &zero));
+
+	r = ratbag_button_disable(button->lib_button);
+	if (r < 0) {
```

Comment:
Typo and missing error handling:
```suggestion
	if (r < 0) {
		r = ratbagd_device_resync(button->device, bus);
				if (r < 0)
					return r;
		}
		if (r == 0) {
```